test-cases/features/uda_udf.yaml

```diff
@@ -0,0 +1,16 @@
+test_duration: 90
+
+prepare_write_cmd: "cassandra-stress user profile=/tmp/c-s_uda_udf.yaml ops'(insert=1)' cl=QUORUM duration=2m -mode native cql3 -rate threads=10"
+stress_cmd: ["cassandra-stress user profile=/tmp/c-s_uda_udf.yaml ops'(my_avg=1, lua_var_length_counter=33, xwasm_plus=33, xwasm_div=32)' cl=QUORUM duration=30m -mode native cql3 -rate threads=20"]
```

Comment:
It should be a longevity test, much longer with much more data in order to exercise the queries on larger partitions, and much more rows.

Reply:
I'm currently testing this approach by running the uda/udf after a much longer prepare_write.